djstripe/models.py

```diff
@@ -267,7 +271,7 @@ def subscription(self):
 
     # TODO: Accept a coupon object when coupons are implemented
     def subscribe(self, plan, charge_immediately=True, **kwargs):
-        # Convert Plan to stripe_id
+        """Convert Plan to stripe_id."""
```

Comment:
This doc is inherited. Add an ignore comment if necessary and revert (The comment is for the ``isinstance`` block, not the method